OrbitCore/Utils.h

```diff
@@ -290,6 +291,10 @@ inline std::string GetPrettySize(uint64_t size) {
   return absl::StrFormat("%.2f TB", size / TB);
 }
 
+inline double GetMillisecondsFromNanoseconds(uint64_t nanoseconds) {
```

Comment:
Do we really want this method to wrap an `absl` one-liner into another one-liner?

Reply:
if you decide to leave it probably rename to NanosecondsToMilliseconds()

Reply:
Let's just remove it for now. Ideally, I wanted to have time always in one format (nanoseconds for example) and then make a conversion only when show time to the user. In this case, we can localize all the conversion in one place.